configuration/analyzer_config.json

```diff
@@ -1334,6 +1334,23 @@
     },
     "soft_time_limit": 30
   },
+  "X-Force_Exchange": {
```

Comment:
combination of `-` and `_` could be a little tricky. Please remove them: `XForceExchange`